test/functional/helpers.lua

```diff
@@ -438,4 +439,5 @@ return {
   curbufmeths = curbufmeths,
   curwinmeths = curwinmeths,
   curtabmeths = curtabmeths,
+  ffi = ffi,
```

Comment:
Yea. AFAIK we are only using FFI for the OS check here and in another pending test fix. So I can change this to the above if you want.